docs/source/shaders/material/as_toon.rst

```diff
@@ -0,0 +1,483 @@
+    It can take the following values accordingly:
+
+        * *Diffuse Affects Both*
+        * *Split Bump*
+
+-----
+
+Transparency Parameters
+^^^^^^^^^^^^^^^^^^^^^^^
+
+*Transparency*
+    The transparency color, affecting shadow tinting as well.
+
+-----
+
+Contour Parameters
+^^^^^^^^^^^^^^^^^^
+
+*Contour Color*
+    The overall color of the outlines.
+
+*Contour Opacity*
+    The overall opacity of the outlines.
+
+*Contour Width*
+    The width of the outlines on the object.
+
+*Contour Object*
+    Generate the contours based on the object ID.
+
+*Contour Material*
+    Generate contours based on the material ID. Objects with the same material ID would share contours.
+
+*Contour Occlusion*
+    Generate contours based on ambient occlusion.
+
+*Contour Creases*
+    Generate contours based on the creases the object [#]_, outlining sudden changes of the surface.
+
+*Occlusion Threshold*
+    The threshold value for the ambient occlusion based outlining. Low values generate outlines in shorter ray hit distances.
+
+*Crease Threshold*
+    The threshold value for the creases based outlining, where low values resultin outlines for smaller changes in the surface of the object.
+
+-----
+
+Matte Opacity Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Enable Matte Opacity*
+    Parameter that toggles matte holdouts.
+
+*Matte Opacity*
+    Matte opacity scaling factor.
+
+*Matte Opacity Color*
+    Holdout color.
+
+-----
+
+Outputs
+-------
+
+*Output Color*
+    The toon BRDF output color.
+
+*Output Transparency*
+    The resulting transparency color.
+
+*Output Matte Opacity*
+    The matte holdout.
+
+-----
+
+.. _label_as_toon_screenshots:
+
+Screenshots
+-----------
+
+.. thumbnail:: /_images/screenshots/toon/as_toon_docs_render1_ibl.png
+   :group: shots_as_toon_group_A
+   :width: 10%
+   :title:
+
+   Toon shader with IBL. Though you have tonal control, IBL makes is still photo-realistic.
+
+.. thumbnail:: /_images/screenshots/toon/as_toon_docs_render2_ibl.png
+   :group: shots_as_toon_group_A
+   :width: 10%
+   :title:
+
+   Toon shader with IBL and a point light. The pointlight makes the tonal progression much more noticeable.
+   A viewer Fresnel term is attenuating the specular highlights.
+
+.. thumbnail:: /_images/screenshots/toon/as_toon_docs_render3.png
+   :group: shots_as_toon_group_A
+   :width: 10%
+   :title:
+
+   Toon shader with an emissive background, a pointlight, and rim lighting to accentuate the emissive background effect.
+   A viewer Fresnel term is attenuating the specular highlights.
+
+.. thumbnail:: /_images/screenshots/toon/as_toon_docs_render4_aniso.png
+   :group: shots_as_toon_group_A
+   :width: 10%
+   :title:
+
+   Toon shader with emissive background, pointlight, and slightly anisotropic specular highlights.
```

Comment:
Fix everywhere: *point light*

Reply:
@dictoon i fixed all this in another branch which includes other docs - all this will be included in that PR and will be sure to reference this one.